### PR TITLE
[res] PyTorchExamples of Softmax

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce Softmax activation

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #4830 , that was missed from previous activation Ops

**Review by mhs4670go — APPROVED** [review]
LGTM
